app/assets/stylesheets/application.scss

```diff
@@ -111,19 +105,19 @@ color: #999999;
   text-align: center;
 }
 
-ul #data_orbit {
-  animation:slide;
-  pause_on_hover:true;
-  animation_speed:500;
-  navigation_arrows:true;
-  bullets:false;
-}
-
 .panel {
   background: #444444;
   border-radius: 5px;
 }
 
 .vote-arrows {
-  color: white;
+  color: $font-color;
+}
+
+.panel .panel_text {
```

Comment:
Merge rule `.panel .panel_text` with rule on line 108<br>Selector `panel_text` should be written in lowercase with hyphens